### ISSUE TITLE
Load PDF preview in Electron with Angular 7

### ISSUE THREAD

**Issue by manojvictor** [electron/electron#19080]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
 4.1.1
* **Operating System:**
 Windows 10
* **Last Known Working Electron version:**
not sure

### Expected Behavior
While loading a pdf (either local or remote url) Electron (well, underlying Chrome browser) simply downloads it instead of preview.

```
  <webview id="foo" src="http://www.pdf995.com/samples/pdf.pdf" style="width:400px; height:480px;">
    <div class="indicator"></div>
  </webview>
```

### Screenshots
![pdf](https://user-images.githubusercontent.com/1560821/60514861-fef78400-9cf7-11e9-914d-bdf6c58de8f1.JPG)

**Comment by codebytere** [comment]
@manojvictor at present, it's unclear that this is an Electron-specific issue. 

Please provide a standalone test that a tester could run to reproduce the issue that demonstrates Electron specificity. This makes fixing issues go more smoothly: not only does it ensure the right thing is tested, but it also prevents triage from bottlenecking on making testers write new tests for each new issue.

I'm setting the more-information-needed label for the above reasons. After you've responded, please @ me in a followup comment. We will be able to more closely look at your issue once you do so.

**Comment by manojvictor** [comment]
@codebytere  I have created a sample repository https://github.com/manojvictor/webview-pdf
In the index.html I have added the webview code where I'm trying to preview the pdf from remote url. 
`<webview id="foo" src="http://www.pdf995.com/samples/pdf.pdf" style="width:400px; height:480px;">
      <div class="indicator"></div>
    </webview>`
But instead of a preview I get a prompt to download the pdf file.
This happen only for pdf file, if I change the src to any other file its getting previewed.
